content/integrations/hashicorp-vault-keystore.md

```diff
@@ -414,6 +459,164 @@ This tutorial shows how to setup a KES server that uses [Vault's K/V engine](htt
+| `keys`                        | Specify an array of keys which *must* exist on the root KMS for KES to successfully start. KES attempts to create the keys if they do not exist and exits with an error if it fails to create one or more key. KES does not accept any client requests until it completes validation of all specified keys.|
+| `cache`                       | Specify expiration of cached keys in `#d#h#m#s` format. Unexpired keys may be used in the event the KMS becomes temporarily unavailable. <br><br> Entries may be set for `any` key, `unused` keys, or `offline` keys. <br><br> If not set, KES uses values of `5m` for all keys, `20s` for unused keys, and `0s` for offline keys. |
+| `log`                         | Enable or disable output for `error` and `audit` type logging events to the console. |
+| `keystore.vault.endpoint` | The Vault endpoint. For example, `https://127.0.0.1:8200` |
+| `keystore.vault.engine` | The path of the K/V engine. For example, `secrets`. <br><br> If empty, defaults to: `kv` (Vault default).  |
+| `keystore.vault.version` | The K/V engine version, either `v1` or `v2`. `v1` engine is recommended. |
+| `keystore.vault.namespace` | An optional [Vault namespace](https://www.vaultproject.io/docs/enterprise/namespaces/index.html). |
+| `keystore.vault.prefix` | An optional K/V prefix. The server stores keys under this prefix. |
+| `keystore.vault.transit` | Optionally encrypt keys stored on the K/V engine with a Vault-managed key. |
+| `keystore.vault.transit.engine` | The path of the transit engine. For example, `my-transit`. <br><br> If empty, it defaults to: `transit` (Vault default). |
+| `keystore.vault.transit.key` | The key name to use to encrypt entries stored on the K/V engine. |
+| `keystore.vault.approle` | [AppRole credentials](https://www.vaultproject.io/docs/auth/approle.html). |
+| `keystore.vault.approle.namespace` | Optional Vault namespace used just for authentication. A single `/` is an alias for the Vault `root` namespace. |
```

Comment:
another occurrence needing the `just`->`only` and `/` treatment from elsewhere

when I'm done reading I'll go find that and suggest it here

Reply:
```suggestion
| `keystore.vault.approle.namespace` | Optional Vault namespace used only for authentication. For the Vault root namespace, use "/". |
```